terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeList.java

```diff
@@ -209,7 +209,8 @@
  * <ol>
  * <li>Returns the specified locale if defined corresponding codelist.</li>
  * <li>Returns the language part of the specified locale if defined corresponding codelist.</li>
- * <li>Returns the {@code fallbackTo} locale.</li>
+ * <li>Returns the {@code fallbackTo} locale. fallbackTo locale is provided by {@link #fallbackTo fallbackTo} or default
+ * locale using language locale of {@link Locale#getDefault Locale#getDefault}.</li>
```

Comment:
- [ ] Seems not correct.
"default locale using language locale of `Locale#getDefault` " indicate only language locale. (`Locale.getDefault().getLanguage()`) But actually, default locale is `Locale.getDefault()` or fallbacked `Locale.getDefault().getLanguage()`.

Please see back your another change https://github.com/terasolunaorg/terasoluna-gfw/pull/874#discussion_r262028454 .

Reply:
Modified by f774c4b74bf4b36bea576cde31ecee75028ad346